handbook/ce/customer_issues.md

```diff
@@ -0,0 +1,32 @@
+# Filing customer issues
+
+Read [the support overview](index.md) before filing an issue.
+
+## Create a customer issue
+
+Customer support tickets should be translated to GitHub issues. In such cases, [create a
+new issue for the request](https://github.com/sourcegraph/customer/issues/new).
+
+Provide the appropriate context and add a label with the affected customer as `customer/$name`. Once its created, sharing it with the required [team](routing_questions.md).
+If necessary, link to the appropriate JIRA Service Desk ticket or [HubSpot](#find-the-unique-company-url) notes.
+
+### General issues
+
+General issues are those that affect more users than those of a particular deployment. In such cases, create a [new issue for the request](https://github.com/sourcegraph/sourcegraph/issues/new/choose) describing it. If there was a previous [customer issue](##create-a-customer-issue), please link the issue in its description.
+
+Remove any potentially private information (e.g. individual people's names, company names, self-hosted Sourcegraph URLs, repo names, screenshots, etc.)
```

Comment:
private information should be in the linked customer issue. don't remove it, place it there

Reply:
Ill rephrase it, I meant it to ensure no private information is on the general issue filed on `sourcegrpah/sourcegraph`

Reply:
@uwedeportivo updated, please review